packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -98,6 +98,13 @@ async function registerTestView(
       testRunner.showErrorMessage(test)
     )
   );
+  // Run Class Tests command
+  testViewItems.push(
+    vscode.commands.registerCommand(
+      'sfdx.force.test.view.runClassTests',
+      test => testRunner.runSingleTest(test)
```

Comment:
Shelby and I discussed the naming of this method and have decided to rename this method to `runTest`, since this method will run a single test within a test class, or all of the methods within a test class.

Reply:
Cool, that sounds good to me too.

Reply:
Sounds good, that threw me off a bit at first as well. Did you consider a more descriptive name or is `runTest` clear enough ?

Reply:
Decided to go with Alli's suggestion of using runTestorTestClass since it was more informative.